### DISCUSSION TITLE
[Bug] problem after installing 1.3.4. there is no disk space _C

### DISCUSSION THREAD

**Discussion by vmyrosh1** [Chia-Network/chia-blockchain#11464] [Support]
### What happened?

After installing Version 1.3.4, synchronization errors occurred. after rebooting, I noticed that there was no space on the disk. I saw that in the chia\mainnet\db folder, the blockchain_v1_mainnet.sqlite file occupies 122 GB. And now the GUI doesn't open at all. There is no space on the disk at all. How do I fix it?

### Version

1.3.4

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by NTOHPN** [comment]
需要更换更大的硬盘

**Comment by NTOHPN** [comment]
容量更大的硬盘

**Comment by vmyrosh1** [comment]
> 容量更大的硬盘

not funny. the gui is installed on disk -C- 250 GB. and until recently, everything worked. What can be done without replacing the disk. not so dramatically? ( there are disks with free space in the computer)

**Comment by NTOHPN** [comment]
> > 容量更大的硬盘
> 
> not funny. the gui is installed on disk -C- 250 GB. and until recently, everything worked. What can be done without replacing the disk. not so dramatically? ( there are disks with free space in the computer)

你的硬盘明显太小了

**Comment by vmyrosh1** [comment]
it is clear that there is not enough disk space. the question is what to do??? what is the reason for this ? there was enough space for a year. can I reinstall the GUI? or something else?

**Comment by emlowe** [comment]
There are various ways you can move the DB to another disk. You will need another SSD, the good news is any inexpensive consumer SSD over USB works great - recommend 516GB, but even 1TB such disks are very affordable.

**Reply by vmyrosh1** [reply]
> There are various ways you can move the DB to another disk. You will need another SSD, the good news is any inexpensive consumer SSD over USB works great - recommend 516GB, but even 1TB such disks are very affordable.

can I point by point? I understand that I don't have enough memory. what should I do if I can point by point? I have two -1 tb ssds in my computer for plotting.
how to migrate a database? delete CHIA from disk -C and make a copy of the database? or just remove CYDIA and install CHIA on another ssd? how then to link the database? Who can tell me how to solve my problem?

**Reply by emlowe** [reply]
Make sure to STOP chia before doing anything!
Here is one way (symlinks): https://thechiaplot.net/2022/02/12/how-to-move-your-chia-database-to-a-new-disk/
Here is a different way:  https://spacefarmers.io/wiki/guides/farming/movedb

**Reply by vmyrosh1** [reply]
> Обязательно ОСТАНОВИТЕ chia, прежде чем что-либо делать!Вот один из способов (символические ссылки): [https://thechiaplot.net/2022/02/12/how-to-move-your-chia-database-to-a-new-disk /](https://thechiaplot.net/2022/02/12/how-to-move-your-chia-database-to-a-new-disk/) Вот другой способ: https://spacefarmers.io/wiki/guides/farming/movedb

Thanks for the links. I did everything as in the second link. I moved the database to another disk. Now only in Connection Status -- Connected. Further in Status, peak height and .... ERROR everywhere. All plots are in UNDETERMINED statuses. Farming status - Undetermined. TOP right green dots SYNCHRONIZED AND CONNECTED (5). so during the week. Maybe reinstall completely CHIA? I have half of the disks in pools. If you reinstall, then how to connect again to the pools? What do you advise?
